src/crimson/os/seastore/btree/fixed_kv_node.h

```diff
@@ -429,7 +429,9 @@ struct FixedKVNode : ChildableCachedExtent {
       if (foreign_it.get_key() == local_it.get_key()) {
 	// the foreign key is preserved
 	if (!child) {
-	  child = source.children[foreign_it.get_offset()];
+	  auto *source_child = source.children[foreign_it.get_offset()];
+	  assert(source_child != nullptr);
```

Comment:
This assert may fail, the child from source can be nullptr. I hit this assert during my test, which is why I didn't include this in the pointer consolidation pr.

Reply:
I see, this is because lba-tree is not supposed to be fully loaded, right?

Will replace it by a comment.

Reply:
Yeah, that's right.